test/integration/scheduler/preemption_test.go

```diff
@@ -549,4 +551,178 @@ var _ = ginkgo.Describe("Preemption", func() {
 		})
 	})
 
+	ginkgo.Context("When most quota is in a shared ClusterQueue with LendingLimit in a cohort", func() {
```

Comment:
I'm tempted to remove this test.
lendingLimit doesn't significantly change the behavior of preemption.borrowWithinCohort
unit tests for simplified scenarios using both features should be enough.

Reply:
Done.